test/integration/guest_creates_account_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.